images/runtime/php/7.4/base.buster.Dockerfile

```diff
@@ -102,7 +102,7 @@ RUN set -eux; \
 #  - https://docs.microsoft.com/en-us/sql/connect/odbc/linux-mac/installing-the-microsoft-odbc-driver-for-sql-server
 # pecl/sqlsrv, pecl/pdo_sqlsrv requires PHP (version >= 7.3.0)
 RUN set -eux; \
-    if [[ $PHP_VERSION == 7.3.* || $PHP_VERSION == 7.4.* || $PHP_VERSION == 8.0.* ]]; then \
+    if [[ $PHP_VERSION == 7.4.* || $PHP_VERSION == 8.0.* ]]; then \
```

Comment:
nit: how does this `PHP_VERSION` environment variable get set? I see at the top of the file that it's declared as `ENV PHP_VERSION 7.4.28`, but is there a way to make it anything other than `7.4.*`? I'm curious how it could be `8.0.*` within this if-statement.

Reply:
We use `images/runtime/php/template.Dockerfile` when supporting a new verison, so basically we copied it from `template.Dockerfile` and set up the `PHP_VERSION `for it, that's why it's more like a general within this if statement.